Backend/utils/cronJobMap.js

```diff
@@ -37,6 +36,10 @@ async function fetchWithRetry(url, options, retries = 3, delayMs = 1000) {
 
 async function getLocationForArtist(artist) {
     try {
+        //TODO: cleanup
```

Comment:
Please do not commit any console.log statements

Reply:
I had removed them before but had to put them back because of a Prisma invocation error that I had to debug. But Sarah said to put the comment "TODO: cleanup" at the top of every console.log statement I intend to clean up later.

Reply:
Sorry, we should not have any log statements committed. I'm following the coding best practices from the intern site cc: @sarahrzhang

Reply:
Got it, just made commit implementing the changes.